seirmo/apps/_simulation_app.py

```diff
@@ -147,6 +158,10 @@ def update_simulation(self, parameters):
             List of parameter values for simulation.
         """
         data = self.simulate.run(parameters[1:], return_incidence=True)
-        self._fig_plot._fig['data'][1]['y'] = data[:, 1] * parameters[0]
+        self._fig_plot._fig['data'][1]['y'] = data[:, 4] * parameters[0]
```

Comment:
Sorry I didn't realise that before, could you specify in the docs that it returns two plotly.graph_object.Figure instances.